mssqlcli/mssql_cli.py

```diff
@@ -87,6 +85,7 @@ def security_words_found_in(query):
         return False
 
 
+# pylint: disable=bad-super-call
```

Comment:
Let's revert all pylint suppressions. We probably should fix these issues, and should do so holistically in another PR.

Reply:
reverted